frontend/src/pages/riddle/riddleHistory.page.tsx

```diff
@@ -53,20 +55,30 @@ const RiddleHistoryPage = () => {
       title: 'Megoldott riddleök'
     }
   ]
-
   return (
     <CmschPage>
       <Helmet title="Megoldott riddleök" />
       <CustomBreadcrumb items={breadcrumbItems} />
       <Stack direction={['column', 'row']} justify="space-between" align={['flex-start', 'center']}>
         <Heading my={5}>Megoldott riddleök</Heading>
-        <Select value={category} onChange={(e) => setCategory(e.target.value)} w="20rem">
-          {query.data!!.map((c) => (
-            <option value={c.categoryName} key={c.categoryName}>
-              {c.categoryName} ({c.submissions.length} megoldott riddle)
-            </option>
-          ))}
-        </Select>
+        <Stack direction={['column']}>
+          <Select value={category} onChange={(e) => setCategory(e.target.value)} w="20rem">
+            {query.data!!.map((c) => (
+              <option value={c.categoryName} key={c.categoryName}>
+                {c.categoryName} ({c?.submissions?.length} megoldott riddle)
+              </option>
+            ))}
+          </Select>
+          {riddleList && (
+            <Select value={riddleList[index].title} onChange={(e) => setIndex(e.target.options.selectedIndex)} w="20rem">
```

Comment:
Ha nincs megoldott riddle, itt hibát dob, mert riddleList[index] az undefined

Reply:
Ohh, nem tudtam, hogy ezt így is lehet, köszi.